corehq/apps/app_manager/views/forms.py

```diff
@@ -654,6 +669,12 @@ def get_form_view_context_and_template(request, domain, form, langs, messages=me
     if toggles.CUSTOM_ICON_BADGES.enabled(domain):
         context['form_icon'] = form.custom_icon if form.custom_icon else CustomIcon()
 
+    if (
+            toggles.COPY_FORM_TO_APP.enabled(domain) or
+            toggles.COPY_FORM_TO_APP.enabled(request.user.username)
```

Comment:
I think you can use `enabled_for_request` here. `toggles.COPY_FORM_TO_APP.enabled_for_request(request)`